src/many-error/src/error.rs

```diff
@@ -147,17 +147,19 @@ many_error! {
             => r#"Invalid method name: "{method}"."#,
     -1001: InvalidFromIdentity as invalid_from_identity()
             => "The identity of the from field is invalid or unexpected.",
-    -1002: CouldNotVerifySignature as could_not_verify_signature(details)
+    -1002: InvalidToIdentity as invalid_to_identity()
+            => "The identity of the to field is invalid or unexpected.",
+    -1003: CouldNotVerifySignature as could_not_verify_signature(details)
```

Comment:
We don't (yet) have any guarantees for error codes, and I'd like to keep the grouped.

We need a full review of all errors. There's a lot of `ManyError::unknown()` in this PR specifically because we need to catalog them all and review our whole error strategy.